graphql/types/ProductSearch.graphql

```diff
@@ -35,11 +37,23 @@ type ProductSearch {
   The possible values in this field are defined by each search engine.
   """
   searchState: String
-  banners: [SearchBanner] @deprecated(reason: "Use the `banners` query instead.")
+  banners: [SearchBanner]
+    @deprecated(reason: "Use the `banners` query instead.")
   """
   Returns the redirect URL set for the given query.
   """
   redirect: String
+  """
+  Options to ProductSearch.
+  """
+  options: ProductSearchOptions
```

Comment:
Shouldn't this be required? Meaning that this will always exist. 

```suggestion
  options: ProductSearchOptions!
```

Reply:
Thanks! I confirmed that these fields will always exist.